ngraph/cmake/external_onnx.cmake

```diff
@@ -46,23 +47,39 @@ macro(onnx_set_target_properties)
         target_compile_options(onnx PRIVATE -Wno-unused-variable -Wno-unused-parameter)
         target_compile_options(onnx_proto PRIVATE -Wno-unused-variable)
     endif()
+    set_target_properties(onnx_proto PROPERTIES
+    CXX_VISIBILITY_PRESET default
+    C_VISIBILITY_PRESET default
+    VISIBILITY_INLINES_HIDDEN OFF)
```

Comment:
Do we need public visibility?

Reply:
It seems to be needed for shared libprotobuf.
Without such visibility I got linking errors for `protoc`.
Sth like:
```
[ 71%] Linking CXX executable ../../../bin/intel64/Debug/protoc
CMakeFiles/protoc.dir/__/src/google/protobuf/compiler/main.cc.o: In function `google::protobuf::compiler::CodeGenerator::CodeGenerator()':
/home/mbencer/workspace/openvino/build/_deps/ext_protobuf-src/src/google/protobuf/compiler/code_generator.h:68: undefined reference to `vtable for google::protobuf::compiler::CodeGenerator'
CMakeFiles/protoc.dir/__/src/google/protobuf/compiler/main.cc.o: In function `google::protobuf::compiler::js::Generator::Generator()'
...
```